### ISSUE TITLE
purpose of triage worksheet

### ISSUE THREAD

**Issue by mrudrara** [salesforce/cloudsplaining#63]
Hello Team, sorry for asking a very basic question. 

https://github.com/salesforce/cloudsplaining/blob/master/examples/files/iam-triage-example.csv

I was trying to understand the field's `services` and `actions` in the triage worksheet. I assume they refer to the count of services for a given policy or a role and a total number of actions for all the services respectively.

if this is the correct assumption. for some of my policies, they are not reflected correctly. I can share examples once I get confirmation

Thanks for this amazing security assessment tool

**Comment by kmcquade** [comment]
@mrudrara - The "services" and "actions" in the triage worksheet could be better termed as "services affected" and "actions affected". So, a given IAM policy could have permissions across three services, but let's say that the policy leverages resource constraints for all actions except for `s3:GetObject`. In this case, the policy would have `1` under services, and `1` under actions.

Does that make sense?

And by all means, feel free to ask basic questions if needed. This is a new tool and chances are that if you're asking the question, others may be asking themselves the same thing. Glad you are enjoying it; happy to help.

**Comment by mrudrara** [comment]
Thank you, in my environment. I have multiple entries of the same Policy
```
Allow-Invoke-Lambdas,role,Customer,3,16,
Allow-Invoke-Lambdas,role,Customer,5,98,
Allow-Invoke-Lambdas,role,Customer,3,4,
Allow-Invoke-Lambdas,role,Customer,3,4,
Allow-Invoke-Lambdas,role,Customer,3,16,
Allow-Invoke-Lambdas,role,Customer,5,98,
Allow-Invoke-Lambdas,role,Customer,3,4,
Allow-Invoke-Lambdas,role,Customer,3,4,
```
Essentially these are inline policies (having the same name)  attached to various roles but with different Services and Actions
wondering why it is categorized as a role.

**Comment by kmcquade** [comment]
I faced the same issue the other day, so I hear you. We used "role" to describe "Inline Role Policy" because listing "Inline Role Policy", "Inline Group Policy", "Inline User Policy" for every inline policy seemed a bit repetitive - but looking back, it seems like it would have been necessary to keep things clear. 

I'm looking into some refactoring strategies for the Inline policies issue. Let me get back to you with some more details about what we're planning or considering in the refactored state - should have those details in the next week.
